test/FormRangeSpec.tsx

```diff
@@ -0,0 +1,47 @@
+import { render } from '@testing-library/react';
+
+import FormRange from '../src/FormRange';
+import FormGroup from '../src/FormGroup';
+
+describe('<FormRange>', () => {
+  it('should render correctly', () => {
+    const { getByTestId } = render(
+      <FormRange
+        id="foo"
+        name="bar"
+        className="my-control"
+        data-testid="test-id"
+      />,
+    );
+    const element = getByTestId('test-id');
+    element.tagName.toLowerCase().should.equal('input');
+    element.id.should.equal('foo');
+    element.classList.length.should.equal(2);
+    element.classList.contains('form-range').should.be.true;
+    element.classList.contains('my-control').should.be.true;
+    element.hasAttribute('name').should.be.true;
+    element.getAttribute('name').should.equal('bar');
+    element.hasAttribute('type').should.be.true;
+    element.getAttribute('type').should.equal('range');
```

Comment:
Don't need hasAttribute since the value is checked on next line anyway

Reply:
Fixed in the commit https://github.com/react-bootstrap/react-bootstrap/pull/6224/commits/023f9b3e2cffbb507001f992d76bb9cfb4243310